xcube/core/store/typespecifier.py

```diff
@@ -74,6 +75,40 @@ def __eq__(self, other) -> bool:
             return False
         return len(self.flags.symmetric_difference(other_type.flags)) == 0
```

Comment:
I realize that this line isn't part of the PR, but I'd really like to understand why it isn't formulated as `return self.flags == other_type.flags`.

Reply:
You're right, the same result could have been achieved by using '=='. I like this solution though, as it shows the difference to the behaviour of the "is_compatible" / "satisifies" method. However, this might become obsolete, as I really like your suggested change for testing the flags in "satisfies".

Reply:
Obsolete, I'll erase `is_compatible`.

Reply:
I'll use Pont's suggestion which is clearly more pythonic.